demo/index.html

```diff
@@ -10,18 +10,35 @@
 
     <link rel="import" href="../../iron-ajax/iron-ajax.html">
     <link rel="import" href="../cosmoz-tree.html">
+    <link rel="import" href="../cosmoz-default-tree.html">
 
     <style is="custom-style" include="demo-pages-shared-styles">
     </style>
   </head>
   <body>
     <template is="dom-bind" id="demo">
+      <h1>See console</h1>
       <iron-ajax url="tree.json" handle-as="json" last-response="{{ treeJson }}" auto></iron-ajax>
     </template>
 
     <script>
-			// TODO(pasleq): do something with the tree
-			var tree = new Cosmoz.Tree(treeJson);
+      window.addEventListener('WebComponentsReady', function(e) {
```

Comment:
you could also do something like `<iron ajax on-response="handleResponse">` and then just `document.querySelector('#demo').handleResponse = function () { ... }` I think to avoid your two addEventListener calls